Test/lit.site.cfg

```diff
@@ -100,6 +100,13 @@ boogieParams = lit_config.params.get('boogie_params', '')
 if len(boogieParams) > 0:
     boogieExecutable = boogieExecutable + ' ' + boogieParams
 
+# Allow user to turn on batch mode in a way that allows skipping
+# tests that don't support it.
+batchMode = lit_config.params.get('batch_mode', '')
+if batchMode:
```

Comment:
Careful! `batchMode` is still just a string here, and Python considers any non-empty string to be truthy. That's why in the CI for this latest commit you can see by the number of unsupported tests that it's running in batch mode all the time. :)

You probably want to make use of `str2bool` here: https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
Or it will be once I get the right Python packages installed in CI.